OrangeDesignSystem/Sources/OrangeDesignSystem/Modules/About/Internal/LitsItems/AppNews/AppNewsViews.swift

```diff
@@ -34,31 +34,59 @@ struct AppNewsList: View {
 
     var body: some View {
         ScrollView {
-            if releaseDescriptions.isEmpty {
-                Text(°°"modules.about.app_news.no_news")
-            } else {
-                ForEach(releaseDescriptions, id: \.version) { releaseDescription in
-                    AboutReleaaseDescriptionEntry(releaseDescription: releaseDescription)
-                        .padding(.horizontal, ODSSpacing.m)
-                    Divider()
-                }
+            switch viewModel.releaseDescriptions {
+            case .loading:
+                loadingView()
+            case let .loaded(releaseDescription) where !releaseDescription.isEmpty:
+                loadedView(releaseDescription)
+            default: // case .error, case .loaded(let releaseDescription) where releaseDescription.isEmpty:
+                errorView()
             }
+        }.onAppear {
+            viewModel.load()
         }
     }
+
+    // ===========
+    // MARK: Views
+    // ===========
+
+    private func loadingView() -> some View {
+        Text(°°"shared.loading")
+    }
+
+    private func loadedView(_ data: [AboutReleaseDescription]) -> some View {
+        ForEach(data, id: \.version) { item in
+            AboutReleaseDescriptionEntry(releaseDescription: item)
+                .padding(.horizontal, ODSSpacing.m)
+            Divider()
+        }
+    }
+
+    private func errorView() -> some View {
+        Text(°°"modules.about.app_news.no_news")
+    }
 }
 
 // =======================================
 // MARK: - About Release Description Entry
 // =======================================
 
-private struct AboutReleaaseDescriptionEntry: View {
+private struct AboutReleaseDescriptionEntry: View {
 
     // =======================
     // MARK: Stored Properties
     // =======================
 
     let releaseDescription: AboutReleaseDescription
 
+    private var formatedDate: String {
```

Comment:
is not a stored properties but a cumputed this why it is a prévale helper, same for all code